### PR TITLE
Add site logo selection

### PR DESCRIPTION
**Pull request by ArsalanSavand**

### CODE REVIEW

File: app/site/site_settings/site_settings_controller.js [hunk 1]
```diff
@@ -30,9 +30,9 @@ function SiteSettingsController($scope, $rootScope, $state, $stateParams, $mdToa
     $scope.siteSettings = $scope.user.sites[$stateParams.s];
     $scope.dolphinService = DolphinService;
 
-    API.Site.get({ site_id: site },
+    API.Site.get({site_id: site},
```
Comment (amirm3hdi) [inline app/site/site_settings/site_settings_controller.js]: @ArsalanSavand You should not change the current format based on your editor.

**Comment by amirm3hdi** [comment]
Not bad, but I'll make it perfect in another pull.

**Comment by Alir3z4** [comment]
@amirm3hdi This has failed the CI, why did you merged it ?

**Comment by Alir3z4** [comment]
It lowered the coverage, I'll right go ahead

**Pull request closed without merging**